src/test/routes/contact-us.ts

```diff
@@ -0,0 +1,17 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
```

Comment:
what is it? :)

Reply:
no idea. copied it from cookies ;)

Reply:
Copying something without knowing its purpose? 🤯 Lets pretend I didn’t see this. 🧐